mesonbuild/compilers.py

```diff
@@ -1809,6 +1809,14 @@ def get_dependency_gen_args(self, outfile):
     def get_buildtype_args(self, buildtype):
         return rust_buildtype_args[buildtype]
 
+    def build_rpath_args(self, build_dir, rpath_paths, install_rpath):
+        return build_unix_rpath_args(build_dir, rpath_paths, install_rpath)
```

Comment:
... Never mind, this seems to be failing on Travis, but not on my local machine. I'll investigate